common/src/main/java/com/wynntils/core/features/properties/FeatureCategory.java

```diff
@@ -16,6 +16,10 @@ public enum FeatureCategory {
     REDIRECTS,
     TOOLTIPS;
 
+    FeatureCategory() {
+        assert !toString().startsWith("core.wynntils");
```

Comment:
Is there a safer check?

Reply:
Safer as in? We do this with features too. I think this is good enough.